src/pages/about.js

```diff
@@ -70,7 +114,7 @@ module.exports = React.createClass({
         <div className="content-wrapper">
           <img className="mozilla-watermark" alt="Mozilla Logo" src="/assets/images/mozilla.1068965acefde994a71c187d253aca2b.svg"/>
         </div>
-        <video controls poster="/assets/images/donate-video-poster.jpg">
+        <video ref="videoPlayer" controls poster="/assets/images/donate-video-poster.jpg">
```

Comment:
There is an updated way of storing and referencing refs.

`ref={(videoPlayer) => { this.videoPlayer = videoPlayer; }}`

Reply:
Bit more info here: https://reactjs.org/docs/refs-and-the-dom.html

You also reference this with `this.videoPlayer` instead of `this.refs.videoPlayer`